tests/unit/lib/experiments/experiment_tests.py

```diff
@@ -34,7 +34,7 @@ def setUp(self):
         self.mock_authentication_token.user_roles = set()
         self.user = User(
             authentication_token=self.mock_authentication_token,
-            loid="t2_1",
+            loid_="t2_1",
```

Comment:
Depending on how public we feel this API is this is technically a breaking change. That said, it doesn't look like the constructor's parameters are publicly documented and I don't see any usage of this in our services in a quick github search so... :see_no_evil: